src/validator.cpp

```diff
@@ -368,11 +368,11 @@ void Validator::swap(Validator &rhs)
 }
 
 bool pathIsRelative(const std::string &path) {
-    // KRM TODO Not sure if this should be in this file or in a utilities file?
-
+    // TODO Not sure if this should be in this file or in a utilities file?
     // Starting with . or .. in any operating system implies relative path
-    if (path.at(0) == '.') 
-        return true;
+    if(path.size())
+        if (path.at(0) == '.') 
+            return true;
 #ifdef _WIN32
```

Comment:
If it is solely a 64 bit consideration then it should be _WIN64 otherwise _WIN32.

Have I ever mentioned how much I hate #ifdef to exclude code?  I'm going to have to go through the guidelines on this one.

Reply:
Which guidelines does this project follow?  That would be a useful thing to know ... :)  I understand the implications of the two flags (_WIN32/_WIN64), I just wondered whether since this can only be built as a 64 bit library whether the flags should reflect that too. Clearly there's nothing in this part of the code which is specific to 64-bits, only to Windows...

I originally didn't use the preproccessor flags as the code itself remains valid if it's all included.  The only reason I needed to use them here was to be able to generate the unit tests on files whose base path was unknown ... I'd rather not have them there either if it's possible.

Reply:
Um this library can be built as a 32 bit library, we just don't happen to build it as such.

I think it should be possible to remove them as there is nothing there that is strictly platform dependent.

Reply:
Code guidelines can be found in the documentation under `coding standard`.

Reply:
Yep, it's gone now, just tidying up tests to suit.